### DISCUSSION TITLE
Add Webauthn as 2fa alternative/replacement

### DISCUSSION THREAD

**Discussion by Miniontoby** [pelican-dev/panel#85] [Ideas]
Please add [WebAuthn](https://webauthn.io/) as 2fa alternative or as 2fa replacement.

Security keys and mobile devices (or any other hardware key like stuff) are more secure for logging in than the 'normal' 2fa. 

It would possibly be the best to recreate the login dialog from github, since it gives you a menu to choose which verification method you would like to use to login with.

**Comment by lancepioch** [comment]
Thanks, this sounds like a great idea for a plugin!
